src/Controller/TestController.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace App\Controller;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Response;
+ 
+ class TestController extends  AbstractController{
+
+    public function index(){
+        return new Response('Test Redirect');
+    }
+
+ }
```

Comment:
Does this need a closing ' ?>' ?

Reply:
Just checked the other files - and I guess not ;-)

Reply:
Is not necessary to be closed w:)